java/src/main/java/com/google/cloud/dataproc/templates/gcs/README.md

```diff
@@ -162,7 +161,48 @@ bin/start.sh \
 
 ```
 
-## 6. Text To BigQuery
+## 6. GCS To Mongo:
+
+Please download the MongoDb connector spark driver and copy it to GCS bucket location.
+
```

Comment:
Can we add some more info about the connector, probably the hyperlinks to maven repo?